<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="styleMenu">
    <property name="title">
     <string>样式</string>
    </property>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>视角</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="itemMenu">
    <property name="title">
     <string>图形</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="itemMenu"/>
   <addaction name="styleMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="headToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actNewFile">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpenFile">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSaveFile">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actSvgFile">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/savesvg.png</normaloff>:/icon/savesvg.png</iconset>
   </property>
   <property name="text">
    <string>导出Svg</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actEnlarge">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Shift+W</string>
   </property>
  </action>
  <action name="actShrink">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actRotateCCW">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
   </property>
   <property name="text">
    <string>向左旋转</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actRotateCW">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="text">
    <string>向右旋转</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actHelp">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>查看文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut"/>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actMoveLeft">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/left.svg</normaloff>:/icon/left.svg</iconset>
   </property>
   <property name="text">
    <string>左移</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actMoveRight">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/right.svg</normaloff>:/icon/right.svg</iconset>
   </property>
   <property name="text">
    <string>右移</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actMoveUp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/up.svg</normaloff>:/icon/up.svg</iconset>
   </property>
   <property name="text">
    <string>上移</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actMoveDown">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/down.svg</normaloff>:/icon/down.svg</iconset>
   </property>
   <property name="text">
    <string>下移</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actFindandReplace">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>查找和替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actReplace">
   <property name="text">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actSelectAll">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actViewRotateCCW">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
   </property>
   <property name="text">
    <string>视角向左旋转</string>
   </property>
  </action>
  <action name="actViewRotateCW">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="text">
    <string>视角向右旋转</string>
   </property>
  </action>
  <action name="actViewEnlarge">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
   </property>
   <property name="text">
    <string>视角放大</string>
   </property>
   <property name="toolTip">
    <string>视角放大</string>
   </property>
  </action>
  <action name="actViewShrink">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>视角缩小</string>
   </property>
   <property name="toolTip">
    <string>视角缩小</string>
   </property>
  </action>
  <action name="actEditShrink">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
   </property>
   <property name="text">
    <string>图形缩小</string>
   </property>
   <property name="toolTip">
    <string>图形缩小</string>
   </property>
  </action>
  <action name="actEditEnlarge">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
   </property>
   <property name="text">
    <string>图形放大</string>
   </property>
   <property name="toolTip">
    <string>图形放大</string>
   </property>
  </action>
  <action name="actEditRotateCCW">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
   </property>
   <property name="text">
    <string>图形向左旋转</string>
   </property>
  </action>
  <action name="actEditRotateCW">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="text">
    <string>图形向右旋转</string>
   </property>
  </action>
  <action name="actViewMoveLeft">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/left.svg</normaloff>:/icon/left.svg</iconset>
   </property>
   <property name="text">
    <string>视角左移</string>
   </property>
   <property name="toolTip">
    <string>视角左移</string>
   </property>
  </action>
  <action name="actViewMoveRight">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/right.svg</normaloff>:/icon/right.svg</iconset>
   </property>
   <property name="text">
    <string>视角右移</string>
   </property>
   <property name="toolTip">
    <string>视角右移</string>
   </property>
  </action>
  <action name="actViewMoveUp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/up.svg</normaloff>:/icon/up.svg</iconset>
   </property>
   <property name="text">
    <string>视角上移</string>
   </property>
   <property name="toolTip">
    <string>视角上移</string>
   </property>
  </action>
  <action name="actViewMoveDown">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/down.svg</normaloff>:/icon/down.svg</iconset>
   </property>
   <property name="text">
    <string>视角下移</string>
   </property>
   <property name="toolTip">
    <string>视角下移</string>
   </property>
  </action>
  <action name="actEditMoveLeft">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/left.svg</normaloff>:/icon/left.svg</iconset>
   </property>
   <property name="text">
    <string>图形左移</string>
   </property>
   <property name="toolTip">
    <string>图形左移</string>
   </property>
  </action>
  <action name="actEditMoveRight">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/right.svg</normaloff>:/icon/right.svg</iconset>
   </property>
   <property name="text">
    <string>图形右移</string>
   </property>
   <property name="toolTip">
    <string>图形右移</string>
   </property>
  </action>
  <action name="actEditMoveUp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/up.svg</normaloff>:/icon/up.svg</iconset>
   </property>
   <property name="text">
    <string>图形上移</string>
   </property>
   <property name="toolTip">
    <string>图形上移</string>
   </property>
  </action>
  <action name="actEditMoveDown">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/down.svg</normaloff>:/icon/down.svg</iconset>
   </property>
   <property name="text">
    <string>图形下移</string>
   </property>
   <property name="toolTip">
    <string>图形下移</string>
   </property>
  </action>
  <action name="actDelSelectedItem">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>删除选中元素</string>
   </property>
   <property name="toolTip">
    <string>删除选中元素</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSelectFrameCol">
   <property name="text">
    <string>选择边框颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSelectFillCol">
   <property name="text">
    <string>选择填充颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSelectTextCol">
   <property name="text">
    <string>选择文字颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSelectTextFont">
   <property name="text">
    <string>选择文字字体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actNoArrowE">
   <property name="text">
    <string>无箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actArrowE">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/arrow.png</normaloff>:/icon/arrow.png</iconset>
   </property>
   <property name="text">
    <string>箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actOpenArrowE">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/openArrow.png</normaloff>:/icon/openArrow.png</iconset>
   </property>
   <property name="text">
    <string>开放型箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDovetailArrowE">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/dovetailArrow.png</normaloff>:/icon/dovetailArrow.png</iconset>
   </property>
   <property name="text">
    <string>燕尾箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDiaArrowE">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/diaArrow.png</normaloff>:/icon/diaArrow.png</iconset>
   </property>
   <property name="text">
    <string>菱形箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actRoundArrowE">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/roundArrow.png</normaloff>:/icon/roundArrow.png</iconset>
   </property>
   <property name="text">
    <string>圆型箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSolidLine">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/solidLine.png</normaloff>:/icon/solidLine.png</iconset>
   </property>
   <property name="text">
    <string>实线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDashLine">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/dashLine.png</normaloff>:/icon/dashLine.png</iconset>
   </property>
   <property name="text">
    <string>短划线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDotLine">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/dotLine.png</normaloff>:/icon/dotLine.png</iconset>
   </property>
   <property name="text">
    <string>点线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDashDotLine">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/dashDotLine.png</normaloff>:/icon/dashDotLine.png</iconset>
   </property>
   <property name="text">
    <string>点划线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDashDDLine">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/dashDDLine.png</normaloff>:/icon/dashDDLine.png</iconset>
   </property>
   <property name="text">
    <string>短划线-点-点线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actStyleSheet">
   <property name="text">
    <string>打开样式表</string>
   </property>
  </action>
  <action name="actMoveSelectedZUp">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoUp"/>
   </property>
   <property name="text">
    <string>图层上移</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actMoveSelectedZDown">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoDown"/>
   </property>
   <property name="text">
    <string>图层下移</string>
   </property>
   <property name="toolTip">
    <string>图层下移</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actMoveSelectedMaxZUp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/toTop.svg</normaloff>:/icon/toTop.svg</iconset>
   </property>
   <property name="text">
    <string>置于顶层</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actMoveSelectedMaxZDown">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/toBottom.svg</normaloff>:/icon/toBottom.svg</iconset>
   </property>
   <property name="text">
    <string>置于底层</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actAboutUs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="actCheck">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
   </property>
   <property name="text">
    <string>检查</string>
   </property>
  </action>
  <action name="actNoArrowB">
   <property name="text">
    <string>无箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actArrowB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/arrow.png</normaloff>:/icon/arrow.png</iconset>
   </property>
   <property name="text">
    <string>箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actOpenArrowB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/openArrow.png</normaloff>:/icon/openArrow.png</iconset>
   </property>
   <property name="text">
    <string>开放型箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDovetailArrowB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/dovetailArrow.png</normaloff>:/icon/dovetailArrow.png</iconset>
   </property>
   <property name="text">
    <string>燕尾箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actDiaArrowB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/diaArrow.png</normaloff>:/icon/diaArrow.png</iconset>
   </property>
   <property name="text">
    <string>菱形箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actRoundArrowB">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/roundArrow.png</normaloff>:/icon/roundArrow.png</iconset>
   </property>
   <property name="text">
    <string>圆型箭头</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actSaveNewFile">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
